rusk-wallet/src/bin/command.rs

```diff
@@ -236,7 +236,7 @@ pub(crate) enum Command {
         reward: bool,
     },
 
-    // Moonlight transaction commands
+    /// Moonlight transaction commands
```

Comment:
I would keep this as is, else it will concatenate this line to the next one in the help output of clap.
 
```suggestion
    // Moonlight transaction commands
```